homatransport/src/application/WorkloadSynthesizer.cc

```diff
@@ -534,6 +542,16 @@ WorkloadSynthesizer::processRcvdMsg(cPacket* msg)
     mesgStats.transportSchedDelay =  rcvdMsg->getTransportSchedDelay();
     emit(mesgStatsSignal, &mesgStats);
 
+    inet::L3Address srcAddr = rcvdMsg->getSrcAddr();
+    inet::L3Address destAddr = rcvdMsg->getDestAddr();
+
+    int srcId = srcAddr.toIPv4().getDByte(2) * 16 + srcAddr.toIPv4().getDByte(3);
```

Comment:
Add a comment how this id calculation is working

Reply:
Actually this is not working as intended right now due to ipv4configurator assigning random addresses, will get rid of this for now.